.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "core"]
 	path = core
-	url = git@github.com:NYPL-Simplified/server_core.git
+	url = git@github.com:datalogics-tarar/server_core.git
+	branch = feature-add-content-source-enki-core
```

Comment:
I understand why you did this during development, but it needs to switch back before merging into master.